code/modules/clothing/head/helmet.dm

```diff
@@ -252,7 +252,7 @@
 
 /obj/item/clothing/head/helmet/riot/knight/templar
 	name = "crusader helmet"
-	desc = "Deus Vult."
+	desc = "A cheap metal helmet that looks straight out of a poorly-funded documentary about the crusades. Might stop a crude melee weapon, but don't expect much beyond that."
```

Comment:
```suggestion
	desc = "A cheap metal helmet that looks straight out of a poorly-funded documentary about the crusades. Might stop a crude melee weapon. The asbestos-lined padding <b>does</b> provide great protection from fire and acid, however..."
```
As it turns out, you would be correct to stop much beyond that. Truly we owe much to the miraculous properties of asbestos.